### PR TITLE
UX optimalizations

### PR DESCRIPTION
**Pull request by jeroensmeets**

### CODE REVIEW

File: src/components/NumberedInfo.vue [hunk 1]
```diff
@@ -76,5 +78,13 @@ export default {
       transform: translateX(42px);
     }
   }
+
+  button:disabled {
+    background-color: #cecece;
```
Comment (IAfanasovMob) [inline src/components/NumberedInfo.vue]: should it go to `_colors.scss`?

File: src/views/Entrepreneur.vue [hunk 2]
```diff
@@ -225,10 +221,15 @@ export default {
         email: '',
         iban: '',
         address: {},
+        checked: false,
```
Comment (IAfanasovMob) [inline src/views/Entrepreneur.vue]: quiet nondescriptive name (:

File: src/views/Entrepreneur.vue [hunk 3]
```diff
@@ -437,18 +439,38 @@ export default {
...
+      cursor: pointer;
+
+      &::after {
+        position: absolute;
+        bottom: 1.35em;
+        left: 1em;
+        z-index: 98;
+        display: block;
+        padding: 0.3em 1em;
+        font-size: 12px;
```
Comment (IAfanasovMob) [inline src/views/Entrepreneur.vue]: I see mix of `px` and `em`. shouldn't we stick to `rem`?

File: src/views/Entrepreneur.vue [hunk 4]
```diff
@@ -437,18 +439,38 @@ export default {
...
+        color: #fff;
```
Comment (IAfanasovMob) [inline src/views/Entrepreneur.vue]: _colors.scss?

**Comment by IAfanasovMob** [comment]
possibly solves #56

File: src/views/Entrepreneur.vue (hunk 2 above)
Comment (jeroensmeets) [reply src/views/Entrepreneur.vue]: changed

File: src/views/Entrepreneur.vue (hunk 3 above)
Comment (jeroensmeets) [reply src/views/Entrepreneur.vue]: good point, but let's get this PR in before optimizing.

File: src/views/Entrepreneur.vue (hunk 4 above)
Comment (jeroensmeets) [reply src/views/Entrepreneur.vue]: changed.

**Pull request closed without merging**